app/controllers/idv/enter_password_controller.rb

```diff
@@ -20,14 +18,14 @@ def new
       Funnel::DocAuth::RegisterStep.new(current_user.id, current_sp&.issuer).
         call(:encrypt, :view, true)
       analytics.idv_enter_password_visited(
-        address_verification_method: address_verification_method,
+        address_verification_method: idv_session.address_verification_mechanism,
         **ab_test_analytics_buckets,
       )
 
       @title = title
       @heading = heading
 
-      @verifying_by_mail = address_verification_method == 'gpo'
+      @verifying_by_mail = idv_session.address_verification_mechanism == 'gpo'
```

Comment:
[nit] Maybe could be refactored into `idv_session.gpo?`

Reply:
Good idea! Adding `idv_session.verify_by_mail?`

Reply:
Done in 9d97ed744